SoftLayer/CLI/virt/create.py

```diff
@@ -155,196 +143,110 @@ def _parse_create_args(client, args):
+@click.option('--vlan-private', type=click.INT,
+              help="The ID of the private VLAN on which you want the virtual server placed")
+@click.option('--subnet-public', type=click.INT,
+              help="The ID of the public SUBNET on which you want the virtual server placed")
+@click.option('--subnet-private', type=click.INT,
+              help="The ID of the private SUBNET on which you want the virtual server placed")
+@helpers.multi_option('--public-security-group', '-S',
+                      help=('Security group ID to associate with the public interface'))
+@helpers.multi_option('--private-security-group', '-s',
+                      help=('Security group ID to associate with the private interface'))
+@click.option('--wait', type=click.INT,
+              help="Wait until VS is finished provisioning for up to X seconds before returning")
+@click.option('--ipv6', is_flag=True, help="Adds an IPv6 address to this guest")
 @environment.pass_env
 def cli(env, **args):
     """Order/create virtual servers."""
+
     vsi = SoftLayer.VSManager(env.client)
     _validate_args(env, args)
+    create_args = _parse_create_args(env.client, args)
 
-    # Do not create a virtual server with test or export
-    do_create = not (args['export'] or args['test'])
-
-    table = formatting.Table(['Item', 'cost'])
-    table.align['Item'] = 'r'
-    table.align['cost'] = 'r'
-    data = _parse_create_args(env.client, args)
-
-    output = []
-    if args.get('test'):
-        result = vsi.verify_create_instance(**data)
-
-        if result['presetId']:
-            ordering_mgr = SoftLayer.OrderingManager(env.client)
-            item_prices = ordering_mgr.get_item_prices(result['packageId'])
-            preset_prices = ordering_mgr.get_preset_prices(result['presetId'])
-            search_keys = ["guest_core", "ram"]
-            for price in preset_prices['prices']:
-                if price['item']['itemCategory']['categoryCode'] in search_keys:
-                    item_key_name = price['item']['keyName']
-                    _add_item_prices(item_key_name, item_prices, result)
-
-        table = _build_receipt_table(result['prices'], args.get('billing'))
-
-        output.append(table)
-        output.append(formatting.FormattedItem(
-            None,
-            ' -- ! Prices reflected here are retail and do not '
-            'take account level discounts and are not guaranteed.'))
-
-    if args['export']:
-        export_file = args.pop('export')
-        template.export_to_template(export_file, args,
-                                    exclude=['wait', 'test'])
-        env.fout('Successfully exported options to a template file.')
-
-    if do_create:
-        if not (env.skip_confirmations or formatting.confirm(
-                "This action will incur charges on your account. Continue?")):
-            raise exceptions.CLIAbort('Aborting virtual server order.')
```

Comment:
I couldn't see the prompt message: "This action will incur charges on your account. Continue?"

Is it not required now?

Reply:
definitely didn't mean to remove this, thanks.